<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #2E4F4F;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(14, 131, 136);
font: 11pt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="tasksListWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(203, 228, 222);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>191</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #2C3333;
color: #CBE4DE
</string>
    </property>
    <property name="text">
     <string>Kaydet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #2C3333;
color: #CBE4DE
</string>
    </property>
    <property name="text">
     <string>Ekle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size : 24pt;
background : rgb(14, 131, 136);
color : rgb(203, 228, 222);
</string>
    </property>
    <property name="text">
     <string>TaskMaster</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="taskLineEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(203, 228, 222);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="silButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #2C3333;
color: #CBE4DE
</string>
    </property>
    <property name="text">
     <string>Sil</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
